apps/judicial-system/web/src/routes/Court/InvestigationCase/HearingArrangements/HearingArrangements.tsx

```
@@ -316,6 +329,39 @@ const HearingArrangements = () => {
+                  tooltip={
+                    isDistrictCourtUser(user) ? (
+                      <Tooltip
+                        text="Lögmaður sem er valinn hér verður skipaður réttargæslumaður brotaþola í þinghaldi og fær sendan úrskurð rafrænt."
+                        placement="right"
+                      />
+                    ) : null
+                  }
+                />
+                {workingCase.victims.map((victim) => (
+                  <Box component="section" marginBottom={4}>
+                    <BlueBox>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)